### PR TITLE
Fix consume-ability of Slicey.

### PR DESCRIPTION
**Pull request by camjc**
- React is capitalised in the browser so change the reference.
- Make Slicey be consumed from one root node instead of using children.

### CODE REVIEW

File: test/components/alexandria/SliceyComponentTest.js [hunk 1]
```diff
@@ -25,6 +41,38 @@ describe('SliceyComponent', () => {
     expect(component.props.width).to.equal(100);
     expect(component.type).to.equal('svg');
 
-    expect(component.props.children).to.have.length(3);
+    expect(component.props.children).to.have.length(4);
+
+    // Marker and Donut should be undefined.
+    expect(component.props.children[2]).to.equal(undefined);
```
Comment (omgaz) [inline test/components/alexandria/SliceyComponentTest.js]: to.be undefined

File: src/components/alexandria/SliceyComponent.js [hunk 2]
```diff
@@ -1,18 +1,27 @@
...
 import {ROUND, HALF, QUARTER, getPointX, getPointY} from 'components/alexandria/slicey/DataProcessor';
 
 require('styles/alexandria/Slicey.scss');
 
-const SliceyComponent = ({dataset, diameter, children}) => {
+const SliceyComponent = ({dataset, diameter, donut, marker}) => {
   const filteredDataset = _.filter(dataset, (datum) => datum.value > 0);
 
+  let donutEl;
+  if (donut !== undefined) {donutEl = <Donut />;}
```
Comment (omgaz) [inline src/components/alexandria/SliceyComponent.js]: Seeing how this prop should be boolean would we prefer a boolean check here instead. e.g. if (donut)

**Comment by omgaz** [comment]
LGTM :fish_cake:

File: src/components/Main.js [hunk 3]
```diff
@@ -15,10 +13,7 @@ class AppComponent extends React.Component {
     return (
       <div className="index">
         <h1>Slicey</h1>
-        <Slicey dataset={sliceyTestData} diameter={150}>
-          <Marker fraction={0.2} />
-          <Donut />
-        </Slicey>
+        <Slicey dataset={sliceyTestData} diameter={150} marker={0.2} donut />
```
Comment (omgaz) [inline src/components/Main.js]: Self closing attributes shouldn't have a trailing slash.

File: src/components/alexandria/SliceyComponent.js (hunk 2 above)
Comment (camjc) [inline src/components/alexandria/SliceyComponent.js]: Good point.

File: src/components/Main.js (hunk 3 above)
Comment (camjc) [inline src/components/Main.js]: See the linked style guide, notably https://github.com/airbnb/javascript/tree/master/react#tags

Comment (omgaz) [inline src/components/Main.js]: Cool.

**Comment by tchcxp** [comment]
LGTM

**Comment by omgaz** [comment]
Merging :ramen:

**Pull request merged**